### ISSUE TITLE
List view buttons missing focus indicators

### ISSUE THREAD

**Issue by twm** [twm/yarrharr#90]
As of https://github.com/twm/yarrharr/commit/b138588bdbaa2e5cde49ad6d9f110c20517bfe73 the toolbar buttons have common focus styling, but the read/unread toggle button and external link on the list views lack it.
